### PR TITLE
feat: Add docker image on artifacts generation summary

### PR DESCRIPTION
**Pull request by toninis**
#### Summary

- Remove monorepo conditions
- Add docker image on artifacts workflow
 
#### Release Note

```release-note

```

### CODE REVIEW

File: .github/workflows/server-ci-artifacts.yml [hunk 1]
```diff
@@ -89,22 +67,24 @@ jobs:
...
         uses: docker/login-action@3da7dc6e2b31f99ef2cb9fb4c50fb0971e0d0139 # v2.1.0
         with:
           username: mattermostdev
           password: ${{ secrets.DOCKERHUB_DEV_TOKEN }}
-        if: needs['determine-if-monorepo'].outputs.is_monorepo == 'true'
+
+      - name: cd/setup-cosign
+        uses: sigstore/cosign-installer@11086d25041f77fe8fe7b9ea4e48e3b9192b8f19 # v3.1.2
+        with:
+          cosign-release: v${{ env.COSIGN_VERSION }}
```
Comment (saturninoabril) [inline .github/workflows/server-ci-artifacts.yml:87]: I may have missed but I didn't see where cosign is applied/used. Is this preparatory setup only? Btw, did we have a prior approval/agreement from SecOps to proceed in using cosign?

Comment (toninis) [reply .github/workflows/server-ci-artifacts.yml:87]: I am just using it to get the digest from Dockerhub . I am not signing anything 😄

**Review by spirosoik — APPROVED** [review]
LGTM

**Review by isacikgoz — APPROVED** [review]

**Pull request merged**